src/js/components/List/List.js

```diff
@@ -132,6 +132,7 @@ const List = React.forwardRef(
       step = paginate ? 50 : undefined,
       onClickItem,
       onMore,
+      orderAlign,
```

Comment:
Thinking on naming here. Also wondering if we are committing to something quite narrow here. What if I want the ability to align my "up/down" controls? Should we be considering that case at this time?

Reply:
Are callers going to want to be able to affect any other Box props?

Would something like
```
order?: { index?: { align?: AlignType } }
```
provide more flexibility long term? This interface could be extended to allow for additional styling such as allowing for pad or allowing for control of the up/down controls, without crowding the top-level List API. 

Wondering aloud here to solicit others thoughts...

Reply:
It's a good point, I think if we plan to enhance to support control alignment in the future, then starting with a more flexible prop name would be valid. I was torn between simplicity with the given use case vs flexibility.

Reply:
The tension is real 😉

Reply:
Yes I also was thinking the same:

I also thought about expanding the `itemProps` to allow for this but my hesitation there was I would assume to be consistant a user would not want to only `align` a certain row however I could be wrong so open to suggestions.. 
otherwise, they would have to do something like  ` itemProps={{ global: { align: 'start' }}}` so that did not feel like the best approach.. 

Also we could do another Prop maybe  `Align table` something and then that would accept the object of the content and the index as well as the controls ..
`AlignTable: {controls: {align: 'start'}}`